js/colors/rock-colors.ts

```diff
@@ -81,7 +81,7 @@ preprocessRockPatterns();
 
 export const getRockPatternImgSrc = (rock: PatternName): string => ROCK_PATTERN[rock].patternImgSrc;
 
-export const getRockCanvasPattern = (ctx: CanvasRenderingContext2D, rock: PatternName) => {
+export const getRockCanvasPattern = (ctx: CanvasRenderingContext2D, rock: PatternName, age: number | undefined = undefined) => {
```

Comment:
Could also use:
```
export const getRockCanvasPattern = (ctx: CanvasRenderingContext2D, rock: PatternName, age?: number) => {
```

Reply:
This change actually should not have been committed at all, as it's not used. I've just removed it here: https://github.com/concord-consortium/tectonic-explorer/commit/099d94e79c90bd61faca283689753f4b6089d72d